src/yang/ietf-access-control-list.yang

```diff
@@ -325,38 +325,39 @@ module ietf-access-control-list {
          interface";
 
       list acl-set {
-        key "set-name type";
+        key "name";
         ordered-by user;
         description
           "List of ingress ACLs on the interface";
 
-        leaf set-name {
+        leaf name {
           type leafref {
             path "../../../../../../acl/name";
           }
           description
-            "Reference to the ACL set name applied on ingress";
+            "Reference to the ACL name applied on ingress";
         }
 
         leaf type {
```

Comment:
This leaf needs to go away. It serves no purpose. acl/name is the key of the list so there is no point in having anything else for referencing an ACL.

Reply:
By "this" you mean the "type" leaf - right? If so, I agree.